tests/projects.spec.ts

```diff
@@ -39,7 +40,9 @@ test("project edit configuration", async ({ page }) => {
     .getByRole("combobox", { name: "Features" })
     .selectOption("customised");
   await page.locator("span").filter({ hasText: "Networks" }).click();
-  await page.locator("label").filter({ hasText: "Network zones" }).click();
+  if (lxdVersion !== "5.0-stable") {
+    await page.locator("label").filter({ hasText: "Network zones" }).click();
+  }
```

Comment:
Shall we add two branches?
In 5.0 ensure the "Network zones" are hidden, in other click them.

Reply:
sure, will add an extra condition to check for 5.0-stable